be/src/pipeline/exec/scan_operator.cpp

```diff
@@ -190,8 +190,7 @@ Status ScanLocalState<Derived>::_normalize_conjuncts(RuntimeState* state) {
             RETURN_IF_ERROR(_normalize_predicate(conjunct->root(), conjunct.get(), new_root));
             if (new_root) {
                 conjunct->set_root(new_root);
-                if (_should_push_down_common_expr() &&
-                    vectorized::VExpr::is_acting_on_a_slot(*(conjunct->root()))) {
+                if (_should_push_down_common_expr(conjunct->root())) {
```

Comment:
pushing down expressions in the agg/mor table may cause more expression calculations. Have you do any benchmark?

Reply:
only push down match predicate, now